packages/android-app/package.json

```diff
@@ -1,19 +1,23 @@
-{  
+{
   "name": "@thetimes/android-app",
   "version": "0.0.1",
+  "private": true,
   "license": "BSD-3-Clause",
   "scripts": {
-    "bundle": "haul bundle --platform android --dev false --entry-file js/index.android.js --bundle-output assets/js/index.android.bundle --assets-dest assets/res/",
+    "bundle:deps": "cd android && ./gradlew generateReactArchives && cd ..",
+    "bundle:js": "haul bundle --platform android --dev false --entry-file js/index.android.js --bundle-output assets/js/index.android.bundle --assets-dest assets/res/",
+    "build": "yarn bundle:deps && yarn bundle:js",
```

Comment:
I'd either call this script `bundle` or the other ones `build:*`

Reply:
I'll call them all build:* to utilise lerna build :+1: